packages/playground/src/components/node_selector/TfNodeDetailsCard.vue

```diff
@@ -279,11 +284,24 @@ export default {
       return imageUrl;
     });
 
-    onMounted(async () => {
+    async function mounted() {
```

Comment:
Maybe we can rename this function to something more meaningful and related to discounts?